scripts/dp_stats/__init__.py

```diff
@@ -1,6 +1,6 @@
 __author__ = 'Sijie, Hafiz'
 
-from .dp_stats import *
-from .dp_svm import *
-from .dp_lr import *
-from .dp_pca import *
\ No newline at end of file
+# from .dp_stats import *
+# from .dp_svm import *
+# from .dp_lr import *
+# from .dp_pca import *
```

Comment:
Be sure to remove code you've already commented out. We're using git, so you can always find the code later if you  change your mind.